src/Sentry/SentrySdk.cs

```diff
@@ -33,7 +33,8 @@ internal static IHub InitHub(SentryOptions options)
             // Side-effects in a factory function 🤮
             options.SetupLogging();
 
-            options.SetupStartupTime();
+            ProcessInfo.Instance ??= new ProcessInfo(options);
+            ProcessInfo.Instance.SetupStartupTime();
```

Comment:
Best to call `SetupStartupTime` only if a new instance was created. Also best to use the local reference to call it since, in theory, `ProcessInfo.Instance` could be null between assigning the reference to it and calling `SetupStartupTime` in the event someone called SentrySdk.Close()` or any other place in the future.

```suggestion
            if (ProcessInfo.Instance is null)
            {
            	var processInfo = new ProcessInfo(options);
            	processInfo.SetupStartupTime();
            	ProcessInfo.Instance = processInfo;
          	}
```